sdk/cognitivelanguage/Azure.AI.Language.Conversations/README.md

```diff
@@ -606,10 +606,10 @@ This project has adopted the [Microsoft Open Source Code of Conduct][code_of_con
 [conversationanalysis_client_class]: https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/cognitivelanguage/Azure.AI.Language.Conversations/src/ConversationAnalysisClient.cs
 [conversationanalysis_client_src]: https://github.com/Azure/azure-sdk-for-net/tree/main/sdk/cognitivelanguage/Azure.AI.Language.Conversations/src/
 [conversationanalysis_samples]: https://github.com/Azure/azure-sdk-for-net/tree/main/sdk/cognitivelanguage/Azure.AI.Language.Conversations/samples/
-[conversationanalysis_nuget_package]: https://www.nuget.org/packages/Azure.AI.Language.Conversations/1.0.0-beta.1
+[conversationanalysis_nuget_package]: https://www.nuget.org/packages/Azure.AI.Language.Conversations/
```

Comment:
Why remove the version?

Reply:
1. Why is it needed. The page has links to all versions.
2. Because someone will have to remember to update it *every* time we release a new version.

Reply:
I can submit a new PR if that's what you want, but I'll assume this extra mundane work - which we also don't seem to do in some other SDKs I checked - is desirable. :)